Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -421,6 +439,9 @@ - (void)initSessionWithLaunchOptions:(NSDictionary *)options isReferrable:(BOOL)
 
 
 - (void)initSessionWithLaunchOptions:(NSDictionary *)options isReferrable:(BOOL)isReferrable explicitlyRequestedReferrable:(BOOL)explicitlyRequestedReferrable automaticallyDisplayController:(BOOL)automaticallyDisplayController {
+    // Log this early. Not consistently showing up when logged from [Branch initialize].
+    [self.class addBranchSDKVersionToCrashlyticsReport];
```

Comment:
Early initialization stuff is tricky. Crashlytics probably isn't fully initialized yet.  

I want to get this PR in for the next release.  We can play around with earlier initialization stuff  later.